media-server/src/main/resources/application.yml

```diff
@@ -0,0 +1,6 @@
+server:
+  port: 9003
+
```

Comment:
Please add the eureka client configuration here. And rename the MEDIA-SERVER to MEDIA-SERVICE